src/System/Taffybar/Information/Brightness.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE OverloadedStrings #-}
+module System.Taffybar.Information.Brightness (readBrightnessValues) where
+
+import System.UDev.Context
+import System.UDev.Device
+import System.UDev.Monitor
+
+import System.UDev.Enumerate
+import System.UDev.List
+
+import Control.Concurrent (Chan, newChan, writeChan, threadWaitRead)
```

Comment:
I try to avoid any specific name imports. If you are worried about name collisions I prefer a qualified import e.g.
``` haskell
import qualified Control.Concurrent as C
```